infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServerCmdLine.java

```diff
@@ -186,6 +187,8 @@ private CorfuServerCmdLine() {
                     + "              LR's cache max number of entries.\n                              "
                     + " --max-replication-data-message-size=<msg-size>                                       "
                     + "              The max size of replication data message in bytes.\n   "
+                    + " --max-replication-write-size=<max-replication-write-size>"
+                    + "              Max size written by the SINK in a single corfu transaction.  Integer.MAX_VALUE by default\n "
```

Comment:
nit: can be reworded to something like - 'max size of replicated data written by ...'

Reply:
done, have included "replicated data " in the param description.

Also mentioned about the testing in the PR description